### ISSUE TITLE
Aliases for syntax highlighting

### ISSUE THREAD

**Issue by kovidgoyal** [sharkdp/bat#230]
I'd like the ability to specify aliases for syntax highlight, so that I could tell bat to highlight file.foo using the syntax highlighter for file.bar instead. Ideally, this would be a config file entry or failing that a command line switch so I can use it via alias.

Thanks.

**Comment by domcorvasce** [comment]
Btw, you can append `-l [lang]` to the command and it'll do what you want:

```shell
bat foo.bar -l ruby
```

**Comment by sharkdp** [comment]
Exactly, `--language`/`-l` already exists and there is already an open ticket for custom extension mappings (#169).

**Comment by kovidgoyal** [comment]
`-l` does not work since that would mean I'd have to wrap bat in my own script to set `-l` based on file type, which is unnecessary overhead. 

Apologies for missing the already open ticket.

**Comment by kovidgoyal** [comment]
Oh and `-l` will not work for catting multiple files.

**Comment by sharkdp** [comment]
Then please explain in detail what you want to achieve. I don't understand.

**Comment by kovidgoyal** [comment]
I want to do the following:

```
bat --aliases ext1:other_ext1,ext2:other_ext2 file1.ext1 file2.ext2
```

this highlight file1.ext1 as if its filename was file1.other_ext1 and file2.ext2 as if its filename was file2.other_ext2

**Comment by sharkdp** [comment]
Okay. This is exactly what #169 is about. Please feel free to comment in that ticket if you have any additional ideas.

**Comment by sharkdp** [comment]
Released in [v0.8.0](https://github.com/sharkdp/bat/releases/tag/v0.8.0)
